src/index.tsx

```diff
@@ -55,11 +55,11 @@ export type Component<T> = React.ComponentClass<T> | React.StatelessComponent<T>
 
 export interface QueryProps {
   containerQuery: Params;
-};
+}
 
 export function applyContainerQuery<T>(
   Component: Component<T & QueryProps>,
-  query: Query,
+  query?: Query,
   initialSize?: Size
 ): React.ComponentClass<T> {
```

Comment:
I think the returned component should have `T & EnhancedProps` instead of `T`, since we are adding `transformQuery` props to the allowed props.